pom.xml

```diff
@@ -144,7 +141,83 @@ under the License.
         <skip.rya.it>true</skip.rya.it>  <!-- modified by  -P enable-it  -->
     </properties>
     
+    <!-- Enable this profile if you want to include Geo functions within Rya. "mvn ... -P geoindexing" -->
     <profiles>
+        <profile>
+            <id>geoindexing</id>
+            <properties>
+                <geomesa.version>1.3.0-m1</geomesa.version> <!-- Newest: 1.3.0-m1 -->
+                <geowave.version>0.9.3</geowave.version> <!-- Newest: 0.9.3 -->
+                <jts.version>1.13</jts.version>
+            </properties>
+            <dependencyManagement>
```

Comment:
I'm not sure how I feel about this.  On the one hand, it does a nice job of separating out geoindexing specific dependencies out of the dependencyManagement section.  But this causes all sorts of eclipse pom parsing errors.  These eclipse errors can be resolved by Right-Clicking the maven project, say rya.geo.common, in the Package Explorer: Maven > Select Maven profiles... and checking geoindexing.  So that part is trivial enough.  My larger concern is we might introduce a different tree of transitive dependencies by changing the project's dependencyManagement in this way, which could cause subtle transitive dependency versioning issues.  IE.  a non-geo function works one way if built with geoindexing profile, and works another way if not built with the geoindexing profile.  Maybe @amihalik can weigh in on this.

Reply:
I'm not worried about the eclipse problem since as you said, you can trivially tell eclipse to resolve using the geoindexing profile.

The transitive dependency problem exists regardless of how dependency management is defined since the management section only defines dependencies that CAN be used, not the dependencies that ARE used. Actual dependencies are required to change that. If we're pulling down different transitive dependencies between geo on and geo off, that's already happening sans this patch.